jedi/api/environment.py

```diff
@@ -32,7 +32,8 @@ class InvalidPythonEnvironment(Exception):
 class _BaseEnvironment(object):
     @memoize_method
     def get_grammar(self):
-        version_string = '%s.%s' % (self.version_info.major, self.version_info.minor)
```

Comment:
BTW: I didn't mention this previously (now I do), but I think it's ok to have a line length of 100 characters occasionally (PEP8 thinks so too), I think these two examples are more readable if they are longer, especially the second one.

Reply:
Ok, please add it to the config then.

Reply:
To which config?

Reply:
`setup.cfg`:

```ini
[flake8]
max-line-length = 100
```